src/java/htsjdk/tribble/util/ParsingUtils.java

```diff
@@ -168,6 +162,30 @@ public static String join(String separator, String[] strings, int start, int end
 
 
     /**
+     * Split the string into tokens separated by the given delimiter. This looks
+     * suspiciously like what String.split should do. It is here because
+     * String.split has particularly poor performance for this use case in version
+     * 6 of the Java standard library.
```

Comment:
Please give the reason for poor performance. I assume that it's because it uses regexes:
String.split(String regex)